static/docs/use-cases/huge-data-on-an-external-drive.md

```diff
@@ -0,0 +1,156 @@
+# Huge Data On An External Local Drive
+
+Sometimes the data may be huge and they are stored on an external local drive.
+By "huge" we mean that they won't fit on our home directory, and even if they
+did, it would certainly take a long time to copy them back an forth from the
+external drive to our home directory. For example let's say that the data are
+stored on an external HDD drive of size 16TB, which is mounted on `/mnt/data/`,
+while the disk of our home directory has a size of only 320GB.
+
+In this case we would like to process the data where they are (on the external
+drive), to save the results there, and certainly to store the cached files on
+the external drive too.
+
+The most easy way to do this would be to locate the <abbr>workspace</abbr> on
+the external drive as well, which could be done like this:
+
+```dvc
+$ sudo su
+# cd /mnt/data/
+# git init
+# dvc init
+```
+
+But in case this is not possible (or is not preferable), we can easily setup the
+<abbr>workspace</abbr> on our home directory, while all the data files and their
+caches keep staying on the external drive. DVC will still be able to track them
+properly.
+
+## Make the data directory accessible
+
+For this to work, first you have to make sure that you can read and write the
+data directory `/mnt/data/`. The most straightforward way to do this is by
+setting proper ownership and permissions to it, like this:
+
+```dvc
+$ sudo chown <username>: -R /mnt/data/
```

Comment:
what about Windows? does it work that way?

Reply:
No, maybe just add a note before this code block linking to https://docs.microsoft.com/en-us/windows-server/manage/windows-admin-center/configure/user-access-control or some other link.

Reply:
Adding.